gazebo_ros/launch/gzserver.launch.py

```diff
@@ -0,0 +1,126 @@
+        DeclareLaunchArgument('record', default_value='false',
+                              description='Set "true" to record state data.'),
+        DeclareLaunchArgument('record_encoding', default_value='',
+                              description='Specify compression encoding format for log data '
+                                          '(zlib|bz2|txt).'),
+        DeclareLaunchArgument('record_path', default_value='',
+                              description='Absolute path in which to store state data.'),
+        DeclareLaunchArgument('record_period', default_value='',
+                              description='Specify recording period (seconds).'),
+        DeclareLaunchArgument('record_filter', default_value='',
+                              description='Specify recording filter '
+                                          '(supports wildcard and regular expression).'),
+        DeclareLaunchArgument('seed', default_value='',
+                              description='Set "true" to start with a random number seed.'),
+        DeclareLaunchArgument('iters', default_value='',
+                              description='Specify number of iterations to simulate.'),
+        DeclareLaunchArgument('minimal_comms', default_value='false',
+                              description='Set "true" to reduce TCP/IP traffic output.'),
+        DeclareLaunchArgument('profile', default_value='',
+                              description='Specify physics preset profile name from the options '
+                                          'in the world file.'),
+        DeclareLaunchArgument('gdb', default_value='false',
+                              description='Set "true" to run gzserver with gdb'),
+        DeclareLaunchArgument('valgrind', default_value='false',
+                              description='Set "true" to run gzserver with valgrind'),
+
+        ExecuteProcess(
+            cmd=['gzserver',
+                 LaunchConfiguration('world'),
+                 _command('version'),
+                 _command('verbose'),
+                 _command('help'),
+                 _command('pause'),
+                 _arg_command('physics'), LaunchConfiguration('physics'),
+                 _arg_command('play'), LaunchConfiguration('play'),
+                 _command('record'),
+                 _arg_command('record_encoding'), LaunchConfiguration('record_encoding'),
+                 _arg_command('record_path'), LaunchConfiguration('record_path'),
+                 _arg_command('record_period'), LaunchConfiguration('record_period'),
+                 _arg_command('record_filter'), LaunchConfiguration('record_filter'),
+                 _arg_command('seed'), LaunchConfiguration('seed'),
+                 _arg_command('iters'), LaunchConfiguration('iters'),
+                 _command('minimal_comms'),
+                 '-s', 'libgazebo_ros_init.so',
+                 '-s', 'libgazebo_ros_factory.so',
```

Comment:
You could expose a parameter like `server-plugin` for the user to pass extra system plugins

Reply:
Same problem as in `gzclient` that users cannot assign a list to an argument